src/content/pages/programme/selection.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Selection Process
+subtitle: How we select talks for EuroPython 2025
+---
+# Selection Process
+To ensure our programme reflects our diverse community, submissions go through up to four rounds of review and refinement before the final programme is announced:
+
+* **NEW! Feedback on proposal (first 100 proposals submitted only):**
+The first 100 proposals will be reviewed by the programme team, and feedback for any possible improvements will be given. This will be done within the CfP submission window to allow you to make changes to your proposal. Please note that this is a **one-off** review and is distinct from the mentoring program.
+* **Community voting:**
+Open to all 2015–2025 EuroPython conference ticket holders and anyone who proposes a talk for the 2025 conference. This helps us judge the uniqueness, specificity, popularity, and appeal of each proposal.
+* **Panel feedback:**
+A diverse group of experienced reviewers refines and provides constructive feedback on submissions within their realm of expertise. This helps us judge the balance, quality, and diversity of our programme.
```

Comment:
Are you intending to share the reviews with the submitters? I think "constructive feedback" suggests that way. If not, maybe changing "feedback" to "input" (open to other ideas) might soften the implication that comments will be shared externally.

Reply:
Hmmm how about something like this? 
 `A diverse group of experienced reviewers evaluates submissions within their area of expertise, offering insights that help the programme team assess the balance, quality, and diversity of the programme.`

Reply:
loved it!